CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
+* bigtable: added `data_boost_isolation_read_only` and `data_boost_isolation_read_only.compute_billing_owner` fields to `google_bigtable_app_profile` resource ([#7789](https://github.com/hashicorp/terraform-provider-google-beta/pull/7789))
+* cloudfunctions: added `build_service_account` field to `google_cloudfunctions_function` resource ([#7713](https://github.com/hashicorp/terraform-provider-google-beta/pull/7713))
+* compute: added `aws_v4_authentication` field to `google_compute_backend_service` resource ([#7775](https://github.com/hashicorp/terraform-provider-google-beta/pull/7775))
+* compute: added `custom_learned_ip_ranges` and `custom_learned_route_priority` fields to `google_compute_router_peer` resource ([#7727](https://github.com/hashicorp/terraform-provider-google-beta/pull/7727))
+* compute: added `export_policies` and `import_policies` fields  to `google_compute_router_peer` resource ([#7748](https://github.com/hashicorp/terraform-provider-google-beta/pull/7748))
+* compute: added `shared_secret` field to `google_compute_public_advertised_prefix` resource ([#7767](https://github.com/hashicorp/terraform-provider-google-beta/pull/7767))
+* compute: added `storage_pool` under `boot_disk.initialize_params` to `google_compute_instance` resource ([#7787](https://github.com/hashicorp/terraform-provider-google-beta/pull/7787))
+* compute: changed `target_service` field on the `google_compute_service_attachment` resource to accept a `ForwardingRule` or `Gateway` URL. ([#7736](https://github.com/hashicorp/terraform-provider-google-beta/pull/7736))
+* container: added documentation for `reservation_affinity` field of `google_container_node_pool` resource ([#7741](https://github.com/hashicorp/terraform-provider-google-beta/pull/7741))
+* container: added field `ray_operator_config` for `resource_container_cluster` ([#7795](https://github.com/hashicorp/terraform-provider-google-beta/pull/7795))
+* datastream: added `merge` and `appendOnly` fields to `google_datastream_stream` resource ([#7726](https://github.com/hashicorp/terraform-provider-google-beta/pull/7726))
+* datastream: promoted `source_config.sql_server_source_config` and `backfill_all.sql_server_excluded_objects` fields in `google_datastream_stream` resource from beta to GA ([#7729](https://github.com/hashicorp/terraform-provider-google-beta/pull/7729))
+* datastream: promoted `sql_server_profile` field in `google_datastream_connection_profile` resource from beta to GA ([#7729](https://github.com/hashicorp/terraform-provider-google-beta/pull/7729))
+* dlp: added `cloud_storage_target` field to `google_data_loss_prevention_discovery_config` resource ([#7734](https://github.com/hashicorp/terraform-provider-google-beta/pull/7734))
+* netapp: added `default_from_api = true` attribute to `smb_settings` field of GoogleCloudNetappVolumes ([#7770](https://github.com/hashicorp/terraform-provider-google-beta/pull/7770))
```

Comment:
I think this change is to make the field O+C to fix a permadiff, which seems a bug fix.